packages/components/src/components/form/form.tsx

```diff
@@ -15,7 +15,8 @@ type ValidationPattern =
   | 'tooShort'
   | 'typeMismatch'
   | 'valid'
-  | 'valueMissing';
+  | 'valueMissing'
+  | ValidationFunction;
```

Comment:
I think there can be an error in `ControlValidationPatterns` so basically you can update and use `ControlValidationPatterns` after:
` type ControlValidationPatterns = ValidationPattern | ValidationFunction;`
What do you think?

Reply:
Yes, changed after your updates